src/components/draggable-box/draggable-box.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {render, screen} from '@testing-library/react'
+import {DraggableBox} from './draggable-box'
+import userEvent from '@testing-library/user-event'
+
+describe('Draggable Box', () => {
+  let child = <h1>hello</h1>
+  let handleClose = jest.fn()
+
+  it('should show consultation notes heading, close button and consultation pad components when draggable box is rendered', () => {
```

Comment:
As you are writing a test case for draggable box which is very generic. The test name should not contain consultation.